.gitignore

```diff
@@ -8,4 +8,5 @@ test/cloud_testing/ec2_config.sh
 python/dist/*
 python/python_cvmfsutils.egg-info/*
 python/MANIFEST
-
+.project
```

Comment:
Ah, the changes in this file as well as in the AUTHORS file must be because the changes come from the same branch.  Let's discuss on Monday.